src/textual/app.py

```diff
@@ -426,6 +433,30 @@ def children(self) -> Sequence["Widget"]:
         except ScreenError:
             return ()
 
+    @contextmanager
+    def batch_update(self) -> Generator[None, None, None]:
+        """Suspend all repaints until the end of the batch."""
```

Comment:
Perhaps an `Example:` or similar thing in the docstring here to remind folk how to call this?

Reply:
Going to leave that for now. Want to figure out best practices before we flesh out docs.